set_environment.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-export LICENSE_ID=
+export LICENSE_ID='72461aba-4438-11ef-a483-d77ff7d7538e'
```

Comment:
I am not sure you want your license included in a public repo...

Reply:
not at all :-)
fixed in 789e625